spec/system/reward_system_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Reward management', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:employee) { create(:employee) }
+  let!(:reward) { create(:reward) }
+
+  # rubocop:disable RSpec/ExampleLength
+  # rubocop:disable RSpec/MultipleExpectations
+
+  it 'enables me to show rewards list and show detailed reward page' do
```

Comment:
I would consider naming it a bit different:
```suggestion
  it 'displays rewards index and shows reward page' do
```

Reply:
Corrected!